tools/releaseBuild/azureDevOps/releaseBuild.yml

```diff
@@ -215,6 +215,49 @@ stages:
             - sign_windows_fxdependent_release
             - sign_windows_fxdependentWinDesktop_release
 
+- stage: PublishSymbol
+  displayName: Publish symbols
+  dependsOn: Build
+  jobs:
+  - job: Symbols_Job
+    pool:
+      name: PowerShell1ES
+      demands:
+      - ImageOverride -equals PSMMS2019-Secure
+    steps:
+    - download: current
+      artifact: results
+      patterns: '**/*.zip'
+
+    - pwsh: |
+        Write-Verbose -Verbose "Enumerating $(Pipeline.Workspace)/results"
+        $downloadedArtifacts = Get-ChildItem -Recurse "$(Pipeline.Workspace)/results"
+        $downloadedArtifacts
+        $expandedRoot = New-Item -Path "$(Pipeline.Workspace)/expanded" -ItemType Directory -Verbose
+        $symbolsRoot = New-Item -Path "$(Pipeline.Workspace)/symbols" -ItemType Directory -Verbose
+        $downloadedArtifacts | Where-Object { $_.Name -like 'PowerShell*-symbols*-signed.zip'} | ForEach-Object {
+          $destFolder = New-Item -Path "$expandedRoot/$($_.BaseName)/" -ItemType Directory -Verbose
+          Expand-Archive -Path $_.FullName -DestinationPath $destFolder -Force
+          Get-ChildItem -Path $destFolder -Recurse -Filter '*.ni.pdb' | ForEach-Object {
+            $runtimeFolder = New-Item -Path "$symbolsRoot/$($destFolder.BaseName)/" -ItemType Directory -Verbose
+            Copy-Item -Path $_.FullName -Destination $runtimeFolder -Verbose
+          }
+        }
+        Write-Verbose -Verbose "Enumerating $symbolsRoot"
+        Get-ChildItem -Path $symbolsRoot -Recurse
+        $vstsCommandString = "vso[task.setvariable variable=SymbolsPath]$symbolsRoot"
+        Write-Verbose -Message "$vstsCommandString" -Verbose
+        Write-Host -Object "##$vstsCommandString"
+      displayName: Expand and capture symbols folders
+    - task: PublishSymbols@2
```

Comment:
The actual publish should happen during the release, not the build.  Otherwise, we may end up publishing symbols for a build we don't release.